tests/check-docs.rs

```diff
@@ -0,0 +1,60 @@
+use regex::Regex;
+use std::{collections::BTreeMap, fmt::Write, path::PathBuf};
+use walkdir::WalkDir;
+
+fn mangle_comments(s: &str) -> String {
+    let result = s
+        .split('\n')
+        .map(str::trim)
+        .filter_map(|line| line.strip_prefix("/// "))
```

Comment:
This removes spacer lines and makes the output look a bit cramped. What do you think about:
```suggestion
        .filter_map(|line| line.strip_prefix("/// ").or_else(line.strip_prefix("///"))
```

Or something similar